corehq/couchapps/attachments/views/attachments/map.js

```diff
@@ -10,6 +10,13 @@ function (doc) {
                     attachments[key] = doc._attachments[key];
                 }
             }
+            for (var key in doc.external_blobs) {
+                if (doc.external_blobs.hasOwnProperty(key) &&
+                    doc.external_blobs[key].content_type !== "text/xml") {
+                    media += 1;
+                    attachments[key] = doc.external_blobs[key];
+                }
+            }
```

Comment:
Unfortunately I don't think there is a way to get around the reindex. There may be a way to decouple it from the rest of the changes though so that you can get it done prior to merging the rest. @dannyroberts any thoughts